shield.go

```diff
@@ -9,27 +9,41 @@ import (
 	"golang.org/x/crypto/pbkdf2"
 )
 
-// Iterations is the number of iterations to run PBKDF2.
-const Iterations = 5000
-
-// MaxLength indicates the maximum length that a password can have.
-const MaxLength = 4096
+var ErrPasswordTooLong = errors.New("password too long")
 
-// SaltLength is the length of the salt generated by GenerateSalt.
-const SaltLength = 32
+// Shield is a struct that contains basic configuration variables used to
+// encrypt and check strings.
+//     iterations: is the number of iterations to run PBKDF2.
+//     maxLength: indicates the maximum length that a password can have.
+//     saltLength: is the length of the salt generated by GenerateSalt.
+type Shield struct {
+	iterations int
+	maxLength  int
+	saltLength int
+}
 
-var ErrPasswordTooLong = errors.New("password too long")
+// Default returns an instance of Shield initialized with default values for
+// each of its variables:
+//     iterations: 5000
+//     maxLength: 4096
+//     saltLength: 32
+func Default() Shield {
+	return Shield{
+		iterations: 5000,
+		maxLength:  4096,
+		saltLength: 32}
+}
```

Comment:
I wouldn't define this as a function, but I'd do what other packages in the stdlib do:

```go
var DefaultShield = Shield{Iterations: 5000, SaltLength: 32}
```